agent/version/formatting.go

```diff
@@ -13,7 +13,11 @@
 
 package version
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-agent/agent/sighandlers/exitcodes"
+)
 
 type Versioner interface {
```

Comment:
Is there a reason to leave this interface around?

Reply:
Versioner interface isn't used anywhere regardless of this change, removing it.